source/website/src/views/Step2.vue

```diff
@@ -68,28 +68,126 @@ SPDX-License-Identifier: Apache-2.0
+                >
+                  <b-form-input id="dataset-description-input" v-model="description" placeholder="(optional)"></b-form-input>
+                </b-form-group>
+                <b-row>
+                  <b-col sm="3">
+                    <b-form-group v-slot="{ ariaDescribedby }">
+                      <slot name="label">
+                        Dataset Type:
+                        <b-link v-b-modal.modal-dataset-type>
+                          <b-icon-question-circle-fill variant="secondary"></b-icon-question-circle-fill>
+                        </b-link>
+                      </slot>
+                      <b-form-radio-group
+                        v-model="dataset_type"
+                        :options="dataset_type_options"
+                        :aria-describedby="ariaDescribedby"
+                        name="dataset-type-radios"
+                        stacked
+                      ></b-form-radio-group>
+                    </b-form-group>
+                  </b-col>
+                  <b-col v-if="dataset_type==='FACT'" sm="3">
+                    <b-form-group v-slot="{ ariaDescribedby }">
+                      <slot name="label">
+                        Dataset Period:
+                        <b-link v-b-modal.modal-dataset-period>
+                          <b-icon-question-circle-fill variant="secondary"></b-icon-question-circle-fill>
+                        </b-link>
+                      </slot>
+                      <b-form-radio-group
+                        id="time_period_options"
+                        v-model="time_period"
+                        :options="time_period_options"
+                        :aria-describedby="ariaDescribedby"
+                        name="time-period-radios"
+                        stacked
+                      ></b-form-radio-group>
+                    </b-form-group>
+                  </b-col>
+                  <b-col sm="2">
+                    Encryption Mode:
+                    <b-link v-b-modal.modal-encryption-mode>
+                      <b-icon-question-circle-fill variant="secondary"></b-icon-question-circle-fill>
+                    </b-link>
+                    <div class="text-muted">
+                      {{ ENCRYPTION_MODE }}
+                    </div>
+                  </b-col>
+                </b-row>
+              </div>
+              <div v-if="dataset_mode === 'JOIN'">
+                <b-row>
+                  <b-col sm="2">
+                    Encryption Mode:
+                    <b-link v-b-modal.modal-encryption-mode>
+                      <b-icon-question-circle-fill variant="secondary"></b-icon-question-circle-fill>
+                    </b-link>
+                    <div class="text-muted">
+                      {{ ENCRYPTION_MODE }}
+                    </div>
+                  </b-col>
+                </b-row>
+              </div>
               <b-row>
```

Comment:
The spacing between encryption mode and the country code drop down does not look right:
<img width="553" alt="image" src="https://user-images.githubusercontent.com/54998167/220442774-d19c2fe6-12a2-4a86-bcc7-a1f187c40c08.png">

Reply:
I used a line break to add extra padding on the bottom of the element - let me know if you think another implementation is cleaner

![Screen Shot 2023-02-21 at 5 50 12 PM](https://user-images.githubusercontent.com/122312553/220500260-b5d799e5-4c78-44f5-a525-dc1ce853d859.png)